app/scripts/controllers/threebox.js

```diff
@@ -73,12 +71,19 @@ class ThreeBoxController {
     }
   }
 
-  async _update3Box ({ type }, newState) {
+  async _update3Box ({ type }, newTypeState) {
     try {
       const { threeBoxSyncingAllowed, threeBoxSynced } = this.store.getState()
       if (threeBoxSyncingAllowed && threeBoxSynced) {
-        await this.space.private.set('lastUpdated', Date.now())
-        await this.space.private.set(type, JSON.stringify(newState))
+        const newState = {
+          preferences: this.preferencesController.store.getState(),
+          addressBook: this.addressBookController.state,
+          lastUpdated: Date.now(),
+          lastMigration: this.lastMigration,
+        }
+        newState[type] = newTypeState
```

Comment:
I don't think we need this line anymore, as we're setting the most current address book and preferences state above anyway.